client/src/bots/bot.ts

```diff
@@ -7,7 +7,7 @@ import { PieceType } from '../model/piece';
 export class Bot {
 	mover: Mover = new Mover();
 	context: { myArmyIndex: number; baseMove: Move } = { myArmyIndex: 0, baseMove: new Move() };
-	depth = 0;
+	depth = 1;
```

Comment:
This will do nothing since the value of depth is overridden in the constructor. It doesn't even have to get an initial value, just a type, as in
depth: number;